app/presenters/publishing_api/speech_presenter.rb

```diff
@@ -0,0 +1,82 @@
+module PublishingApi
+  class SpeechPresenter
+    include UpdateTypeHelper
+
+    attr_accessor :item
+    attr_accessor :update_type
+
+    def initialize(item, update_type: nil)
+      self.item = item
+      self.update_type = update_type || default_update_type(item)
+    end
+
+    def content_id
+      item.content_id
+    end
+
+    def content
+      content = BaseItemPresenter.new(item).base_attributes
+      content.merge!(
+        description: item.summary,
+        details: details,
+        document_type: "speech",
+        public_updated_at: item.public_timestamp || item.updated_at,
+        #TODO: rendering app is hard coded until format is ready
+        #rendering_app: Whitehall::RenderingApp::GOVERNMENT_FRONTEND,
+        rendering_app: "whitehall-frontend",
+        schema_name: "speech",
+      )
+      content.merge!(PayloadBuilder::PublicDocumentPath.for(item))
+      content.merge!(PayloadBuilder::AccessLimitation.for(item))
+      content.merge!(PayloadBuilder::FirstPublishedAt.for(item))
+    end
+
+    def details
+      details = {
+        body: item.body,
+        political: item.political,
+        delivered_on: item.delivered_on.iso8601,
+      }
+      details.merge!(image_payload) if speaker_has_image?
+      details.merge!(PayloadBuilder::PoliticalDetails.for(item))
+    end
+
+    def links
+      links = LinksPresenter.new(item).extract([:organisations])
+      links.merge!(links_for_policies)
+      links.merge!(links_for_speaker)
+      links.merge!(links_for_topical_events)
+    end
+
+  private
+
+    def image_payload
+      {
+        image: {
+          alt_text: speaker.name,
+          url: speaker.image.url,
+        }
+      }
+    end
+
+    def links_for_policies
```

Comment:
I know we had a conversation about this but it looks like everything else is using `related_policies` for this, passed as a key to the `LinksPresenter`